gkShell/apache2/000-default.conf

```diff
@@ -0,0 +1,44 @@
+<VirtualHost *:80>
+        # The ServerName directive sets the request scheme, hostname and port that
+        # the server uses to identify itself. This is used when creating
+        # redirection URLs. In the context of virtual hosts, the ServerName
+        # specifies what hostname must appear in the request's Host: header to
+        # match this virtual host. For the default virtual host (this file) this
+        # value is not decisive as it is used as a last resort host regardless.
+        # However, you must set it for any further virtual host explicitly.
+        #ServerName www.example.com
+
+        ServerAdmin webmaster@localhost
+        DocumentRoot /var/www/html
+
+        # Available loglevels: trace8, ..., trace1, debug, info, notice, warn,
+        # error, crit, alert, emerg.
+        # It is also possible to configure the loglevel for particular
+        # modules, e.g.
+        #LogLevel info ssl:warn
+
+        LogFormat "%h %l %u %t \"%r\" %>s %b \"%{Referer}i\" \"%{User-Agent}i\"" combined
+        LogFormat "%{X-Forwarded-For}i %l %u %t \"%r\" %>s %b \"%{Referer}i\" \"%{User-Agent}i\"" proxy
+        SetEnvIf X-Forwarded-For "^.*\..*\..*\..*" forwarded
+        CustomLog "${APACHE_LOG_DIR}/access.log" combined env=!forwarded
+        CustomLog "${APACHE_LOG_DIR}/access.log" proxy env=forwarded
+
+        ErrorLog ${APACHE_LOG_DIR}/error.log
+
+        # For most configuration files from conf-available/, which are
+        # enabled or disabled at a global level, it is possible to
+        # include a line for only one particular virtual host. For example the
+        # following line enables the CGI configuration for this host only
+        # after it has been globally disabled with "a2disconf".
+        #Include conf-available/serve-cgi-bin.conf
+
+        # Issue-68: Change link to download GeoKrety Toolbox
+        Redirect "/download/GeoKrety.Toolbox.user.js" "https://raw.githubusercontent.com/geokrety/GeoKrety-Toolbox/master/GeoKrety.Toolbox.user.js"
+
+        # Issue-63: help page from generated html files
+        RewriteEngine on
+        RewriteCond %{REQUEST_FILENAME} ^/help.php$
+        RewriteRule help.php$ /help_generate.php?help=en [L]
```

Comment:
Change on this file will conflict with PR #222. Maybe `git` is smart enough to detect that the file has moved and will apply nicely?

Reply:
PR 222 update `docker/apache2` one.
`gkShell/apache2` should be removed soon (replaced by a live resource copy & gitignore )

Reply:
done